ui/main/src/app/api/opfab.api.ts

```
@@ -30,17 +33,17 @@ export class OpfabAPIService {
+        OpfabAPI.translationService = translationService;
     }
 
     public static init() {
-        OpfabAPIService.initCurrentCard();
-        OpfabAPIService.initCurrentUserCard();
+        OpfabAPI.initCurrentCard();
+        OpfabAPI.initCurrentUserCard();
     }
 
     public static initCurrentCard() {
         const self = this;
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)